apps/velo-external-db/test/e2e/app_schema_hooks.e2e.spec.ts

```diff
@@ -139,7 +139,7 @@ describe(`Velo External DB Schema Hooks: ${currentDbImplementationName()}`, () =
                     })
 
                     await expect(axios.post('/schemas/column/remove', { collectionName: ctx.collectionName, columnName: ctx.column.name }, authOwner)).rejects.toMatchObject(
-                        errorResponseWith(400, 'Should not be removed')
+                        errorResponseWith(500, 'Should not be removed')
```

Comment:
the SPI expects here 500? (and in all the others that you've changed in this file ? )
thats weird because it's clearly not 500, maybe we should tell them.

BTW, the test name is 'should throw 400' so if you decide it's really 500 please change the test names.